src/twin_actor.rs

```diff
@@ -66,43 +62,41 @@ impl Actor for TwinActor {
 
         let addr = ctx.address();
 
-        let api_config = self.api_config.clone();
-        let token = self.token.clone();
+        let auth_builder = self.auth_builder.clone();
         let twin = self.twin.clone();
-        let twin_label = twin.label.clone();
```

Comment:
Why has this binding been moved (line 98) ? We use this variable several times in this function so I think it's good to have it here

Reply:
Seeing it there made me curious where it's used so I've decided to move it closer to the first usage to make it more obvious. I guess it's just personal preference. Happy to change if you have strong oppinions.

Reply:
We also use it for `build_twin_properties` which comes before that. Anyway I don't have a strong opinion

Reply:
Ah, I see! Actually I think that we can just remove this declaration and use `twin.label`!